eng/pipelines/aggregate-reports.yml

```diff
@@ -6,6 +6,11 @@ resources:
     branch: main
     tags: 
     - scheduled
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/heads/AddPoliCheckFilter
```

Comment:
I assume this should be `master`?

Reply:
Right. I will have it changed before merging.

Reply:
It should actually be release tag not a branch name.